src/plugins/data_source_management/public/components/direct_query_data_sources_components/direct_query_data_connection/manage_direct_query_data_connections_table.tsx

```diff
@@ -128,50 +130,68 @@ export const ManageDirectQueryDataConnectionsTable = ({
   const fetchDataSources = useCallback(() => {
     setIsLoading(true);
 
-    const fetchConnections = featureFlagStatus
-      ? getDataSources(savedObjects.client)
-      : http.get(`${DATACONNECTIONS_BASE}`);
-
-    return fetchConnections
-      .then((response) => {
-        return featureFlagStatus
-          ? fetchDataSourceConnections(
-              response,
-              http,
-              notifications,
-              true,
-              getHideLocalCluster().enabled
-            )
-          : response.map((dataConnection: DirectQueryDatasourceDetails) => ({
-              id: dataConnection.name,
-              title: dataConnection.name,
-              type:
-                {
-                  S3GLUE: 'Amazon S3',
-                  PROMETHEUS: 'Prometheus',
-                }[dataConnection.connector] || dataConnection.connector,
-              connectionType: dataConnection.connector,
-              description: dataConnection.description,
-            }));
-      })
-      .then((finalData) => {
-        setData(
-          featureFlagStatus
+    const fetchOpenSearchConnections = async (): Promise<DataSourceTableItem[]> => {
+      const fetchConnections = featureFlagStatus
+        ? getDataSources(savedObjects.client)
+        : http.get(`${DATACONNECTIONS_BASE}`);
+
+      return fetchConnections
+        .then((response) => {
+          return featureFlagStatus
+            ? fetchDataSourceConnections(
+                response,
+                http,
+                notifications,
+                true,
+                getHideLocalCluster().enabled
+              )
+            : response.map((dataConnection: DirectQueryDatasourceDetails) => ({
+                id: dataConnection.name,
+                title: dataConnection.name,
+                type:
+                  {
+                    S3GLUE: 'Amazon S3',
+                    PROMETHEUS: 'Prometheus',
```

Comment:
Plz use `DatasourceTypeToDisplayName`

Reply:
change to use `DatasourceTypeToDisplayName.S3GLUE` and `DatasourceTypeToDisplayName.PROMETHEUS`